### DISCUSSION TITLE
What causes this error in my logs

### DISCUSSION THREAD

**Discussion by chianut** [Chia-Network/chia-blockchain#5087] [Support]
Hi there, I am on version 1.1.5, I am currently plotting and farming... I am getting this error in the logs and was wondering if someone can please shed some light on if I am not farming properly and essentially wasting my time and my app isnt working properly?

harvester harvester_server        : ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection {'host': '127.0.0.1', 'port': 8447}. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call
  File "asyncio\tasks.py", line 435, in wait_for
concurrent.futures._base.CancelledError

I am in the process of banging my head against the wall opening port 8444, its not as simple with my set up... ugh, but apart from that I am synched etc and I wasnt sure if the issue was fully related to 8444?

Thanks so much in advance for any advice...

Nathan

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
